.github/workflows/ansible-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: ansible-lint
+      uses: ansible/ansible-lint-action@v4.1.0.post0
+      with:
+        args: "-R -r ../lint-rules"
```

Comment:
we should first git clone our lint rules?

Reply:
Yes, but somehow I was not able to figure out how to do that correctly.